modules/devdocs/deps.edn

```diff
@@ -1,16 +1,11 @@
 {:path ["src"]
- :deps {re-frame/re-frame {:git/url "https://github.com/nextjournal/freerange"
+ :deps {io.github.nextjournal/clerk {:git/sha "8cbb055078eddc2a20a6c9ea5d78155b262a7988"}
+        io.github.nextjournal/dejavu {:git/url "git@github.com:nextjournal/dejavu.git"
```

Comment:
I guess we need to make [dejavu](https://github.com/nextjournal/dejavu) public, since viewers is a dev dependency of Clerk? I don't have enough permissions to do so.

Reply:
It's public now.